addons/main/script_macros.hpp

```diff
@@ -134,4 +134,6 @@
 #define MRAD_TO_DEG(d) ((d) / 17.45329252) // Conversion factor: 9 / (50 * PI)
 #define MOA_TO_RAD(d) ((d) * 0.00029088) // Conversion factor: PI / 10800
 
+#define ZEUS_ACTION_CONDITION (QUOTE(QUOTE(ADDON)) in curatorAddons _target)
```

Comment:
not sure about macro name because double `QUOTE` can't be used in actions added by script not config

Reply:
`"ace_zeus" in curatorAddons z` is ~0.2ms with just ace or 0.5 with my full modset
so we might want to use cached call instead
```
#define ZEUS_ACTION_CONDITION ([nil, {QUOTE(QUOTE(ADDON)) in curatorAddons _target}, missionNamespace, QUOTE(QUOTE(GVAR(zeusCheck))), 60] call ace_common_fnc_cachedCall)
```

Reply:
Maybe add event to clear this cache, e.g. that one on interaction menu closed (don't remember exactly name)? 60s timeout can be not so effective if `removeCuratorAddons`/`addCuratorAddons` are used.

Reply:
`ace_interactMenuClosed`